velero/backup-all-volumes/artifacthub-pkg.yml

```diff
@@ -17,6 +17,27 @@ install: |-
 keywords:
   - velero
   - kyverno
+readme: |
+  # Kyverno Policy: Backup All Volumes
+  This repository contains a Kyverno policy package for automatically backing up all volumes in a Kubernetes cluster. The policy is designed to work with the Kyverno Kubernetes policy management tool.
+  ## Overview
+  The `backup-all-volumes` Kyverno policy helps you ensure that all Persistent Volumes (PVs) and Persistent Volume Claims (PVCs) in your Kubernetes cluster are backed up regularly. This policy enforces a backup configuration for all volumes, improving the reliability and availability of your application data.
+  ## Installation
+  To install the `backup-all-volumes` Kyverno policy, please apply the Kyverno policy to your cluster:
+  ```bash
+    kubectl apply -f https://raw.githubusercontent.com/kyverno/policies/main/velero/backup-all-volumes/backup-all-volumes.yaml
+  ```
+  ## Uninstallation
+  To uninstall the `backup-all-volumes` Kyverno policy from your cluster, run:
+  ```bash
+  kubectl delete -f https://raw.githubusercontent.com/kyverno/policies/main/velero/backup-all-volumes/backup-all-volumes.yaml
+  ```
+  ## License
+  This project is licensed under the [Apache License, Version 2.0](https://github.com/kyverno/policies/blob/main/LICENSE).
+  ## Kyverno annotations
+  Refer to the documentation for more details on Kyverno annotations: https://artifacthub.io/docs/topics/annotations/kyverno/
+  ## Support
+  If you encounter any issues or need help with this policy, please create an issue in this repository or join the [Kyverno Slack](https://kyverno.io/community/) community for support.
```

Comment:
The only thing that should be in this section is the value of the `policies.kyverno.io/description` annotation and the section called Kyverno annotations. In the case of this one, it should look like:

```
In order for Velero to backup volumes in a Pod using an opt-in approach, it
requires an annotation on the Pod called `backup.velero.io/backup-volumes` with the
value being a comma-separated list of the volumes mounted to that Pod. This policy
automatically annotates Pods (and Pod controllers) which refer to a PVC so that
all volumes are listed in the aforementioned annotation if a Namespace with the label
`velero-backup-pvc=true`.

Refer to the documentation for more details on Kyverno annotations: https://artifacthub.io/docs/topics/annotations/kyverno/
```

Follow suit for the rest of the samples here, please.